src/main/java/seedu/address/model/UserPrefs.java

```diff
@@ -100,11 +101,9 @@ public boolean equals(Object other) {
 
         UserPrefs o = (UserPrefs) other;
 
-        return guiSettings.equals(o.guiSettings)
-                && menuFilePath.equals(o.menuFilePath)
-                && ordersFilePath.equals(o.ordersFilePath)
-                && tablesFilePath.equals(o.tablesFilePath)
-                && statisticsFilePath.equals(o.statisticsFilePath);
+        return guiSettings.equals(o.guiSettings) && menuFilePath.equals(o.menuFilePath) && ordersFilePath
+                .equals(o.ordersFilePath) && tablesFilePath.equals(o.tablesFilePath) && statisticsFilePath
+                .equals(o.statisticsFilePath);
```

Comment:
This line should be indented one more level

Reply:
This complies with the Java Coding Standard.